.config/ansible-lint.yml

```diff
@@ -0,0 +1,5 @@
+warn_list:
```

Comment:
Can you move the content of this file from `.config/ansible-lint.yml` to `.ansible-lint`?

Reply:
This is the default location for the config file. The CI action which we are using is picking config from `.config` directory itself.

Reply:
As an alternative, We would want to tweak entire action code and use it directly here.

Reply:
I saw that you named the file `ansible-lint.yml` but to be recognised by Ansible lint, the file should not have an extension and it should begin with a `.` like this `.ansible-lint`. Can you try to rename the file?